### PR TITLE
Abb 191 154

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
One new record, corrects in two others.

### CODE REVIEW

File: ParisBNF/abb/BNFabb191.xml [hunk 1]
```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb191" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+           <title>Commentaries in in Gǝʿǝz on the Gospel of Matthew, Luke, and John, </title>
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb191.xml:7]: delete one of the "in"

File: ParisBNF/abb/BNFabb191.xml [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+                                       cannot be checked on the basis of the available images).</note>
+                                     
+                                     <dimensions type="leaf" unit="mm" xml:id="leafdim2">
+                                       <height>300</height>
+                                       <width>220</width>
+                                     </dimensions>
+                                     <note corresp="#leafdim2">Leaf dimensions for <locus from="138" to="228"/>, according to M. Chaîne.</note>                                                                          
+                                   </extent>                           
+                              
+                                  <foliation>Foliation in the upper right corner of the folia. The first two parchment leave are not foliated (assigned the numbers
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb191.xml:136]: leaves

File: ParisBNF/abb/BNFabb191.xml [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+                                       <material key="paper"/>
+                                   </decoNote>
+                                </binding>
+                             </bindingDesc>
+                          </physDesc>
+                                              <history>
+                                                 <origin>
+                                                   <origDate when="1647">1647</origDate>
+                                                  According to <ref target="#colop1"/>, at least part of the manuscript 
+                                                    can be dated to 1647 CE (=1328 according to the Era of Alexander the Great).
```
Comment (CarstenHoffmannMarburg) [inline ParisBNF/abb/BNFabb191.xml:247]: 'era' according to deepl write.

File: ParisBNF/abb/BNFabb191.xml (hunk 1 above)
Comment (eu-genia) [reply ParisBNF/abb/BNFabb191.xml:7]: and comma in the end

File: ParisBNF/abb/BNFabb191.xml [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+                                       <width>220</width>
+                                     </dimensions>
+                                     <note corresp="#leafdim2">Leaf dimensions for <locus from="138" to="228"/>, according to M. Chaîne.</note>                                                                          
+                                   </extent>                           
+                              
+                                  <foliation>Foliation in the upper right corner of the folia. The first two parchment leave are not foliated (assigned the numbers
+                                    <locus target="#i #ii"/>).</foliation>
+                                  
+                                  <collation>The quire structure cannot be ascertained due to the condition of the manuscript.</collation>
+                                  <condition key="good">The original binding is missing; all quires are dismantled,
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb191.xml:140]: good?

File: ParisBNF/abb/BNFabb191.xml [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+                                     <material key="cardboard"/>
+                                       <material key="paper"/>
+                                   </decoNote>
+                                </binding>
+                             </bindingDesc>
+                          </physDesc>
+                                              <history>
+                                                 <origin>
+                                                   <origDate when="1647">1647</origDate>
+                                                  According to <ref target="#colop1"/>, at least part of the manuscript
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply ParisBNF/abb/BNFabb191.xml:246]: `<ref target="#coloph1">colophon 1</ref>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb191.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb191.xml:7]: ok

File: ParisBNF/abb/BNFabb191.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb191.xml:136]: ok

File: ParisBNF/abb/BNFabb191.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb191.xml:247]: In the Bm predominantly "Era".

File: ParisBNF/abb/BNFabb191.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb191.xml:140]: ok, changed.

File: ParisBNF/abb/BNFabb191.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb191.xml:246]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**